### PR TITLE
Changed the consent screen for GB.

### PR DESCRIPTION
**Pull request by aikewoody**
# Description

https://www.notion.so/joinzoe/New-user-signup-incorporating-reconsent-changes-4a2db938fcbf481b911b90f65a32d4bf

## On what platform have you tested the change?

- [ ] Android - Emulator
- [ ] Android - Physical device
- [x] iOS - Emulator
- [ ] iOS - Physical device

## Out of scope and potential follow-up

[comment]: <> (Are there any related changes that you plan to do in a follow-up PR or voluntarily excluded from the scope?)

## Screenshots

Simulator Screen Shot - iPhone 8 - 2021-09-10 at 11 15 45

Simulator Screen Shot - iPhone 8 - 2021-09-10 at 11 15 48

## Checklist (for submission)

- [ ] Analytics/tracking events have been added (if relevant)
- [ ] Added new screen(s) to `src/features/screens.ts` and new Redux slice(s) to `mocks/mockedInitialState.ts` (if relevant)

## Checklist (for reviewers)

- [ ] Checked against Figma designs (if relevant)
- [ ] Checked that data has been successfully saved to the backend (if relevant)

## Checklist (for merger)

- [ ] Make sure the right merge strategy is chosen, commit merge into master and squash merge into develop.

### CODE REVIEW

**Comment by aikewoody** [comment]
@selvach Do I need to update the version number of the UK consent?

**Comment by selvach** [comment]
Check with Christina if the study consent should change?

File: assets/lang/en.json [hunk 1]
```diff
@@ -60,6 +60,7 @@
     "take-a-minute": "Take 1 minute each day and help fight the outbreak.",
     "disclaimer": "This app allows you to help others, but does \n not give health advice. If you need health \n advice please",
     "disclaimer-link": "visit the NHS website",
+    "disclaimer2": "Moving forward, you may also help detect early symptoms and uncover the role of nutrition and lifestyle in other major diseases",
```
Comment (juliatan) [inline assets/lang/en.json:63]: There should be a full stop at the end of the sentence.

**Review by juliatan — CHANGES_REQUESTED** [review]
Nice one for getting it done so quickly Aike!

Some other things: 
- I think some of the logos on the welcome screen need updating e.g. the ZOE logo should contain the registered trademark.
- On the screen with the legal cards, the title and subtitle is missing.

Simulator Screen Shot - iPhone 11 - 2021-09-03 at 17 17 34
Simulator Screen Shot - iPhone 11 - 2021-09-03 at 17 17 40

File: src/features/register/Welcome2Screen.tsx [hunk 2]
```diff
@@ -55,14 +56,11 @@ const Welcome2Screen: React.FC<TProps> = ({ navigation }) => {
...
   const partnersLogos = React.useMemo(() => {
-    if (isGBCountry()) {
-      return gbPartners;
-    }
     if (isSECountry()) {
       return svPartners;
     }
     return usPartners;
-  }, [isGBCountry(), isSECountry(), gbPartners, svPartners, usPartners]);
+  }, [isGBCountry(), isSECountry(), svPartners, usPartners]);
```
Comment (juliatan) [inline src/features/register/Welcome2Screen.tsx:63]: Guess you can remove `isGBCountry()` from dependency array.

**Review by juliatan — CHANGES_REQUESTED** [review]

**Review by juliatan — DISMISSED** [review]

**Comment by juliatan** [comment]
@aikewoody Looking at Sol's latest designs, the copy has changed 😕

**Comment by juliatan** [comment]
@aikewoody The first screen has also changed and needs updating for all the copy.

Also, this screen needs centre alignment. It also looks like the style of the back navigation button has changed.
Simulator Screen Shot - iPhone 11 - 2021-09-13 at 14 20 27

**Comment by aikewoody** [comment]
@juliatan I have ignored the register screen as it's out of scope for the reconsent changes but you're right that in the designs this screen has changes. Will implement those as it makes the screen look better than it is now. About the back button, I want to keep that as is and create a ticket to replace all back buttons through the app to prevent having two different styles at the same time.

**Review by juliatan — APPROVED** [review]

**Pull request merged**